src/one_dragon/envs/git_service.py

```
@@ -1,454 +1,765 @@
-        log.info(gt('获取依赖文件的最后修改时间'))
-        return cmd_utils.run_command([self.env_config.git_path, 'log', '-1', '--pretty=format:"%ai', '--', self.project_config.requirements])
+            if remote_ref not in repo.references:
+                return False, gt('与远程分支不一致')
+
+            remote_oid = repo.references[remote_ref].target
+            local_oid = repo.head.target
+
+            if local_oid == remote_oid:
+                return True, ''
+
+            # 比较提交是否相同；否则比较树差异
```

Ruff (0.14.1)

Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)